chapters/equations.tex

```diff
@@ -718,14 +718,14 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 There is no special handling of inactive \lstinline!when!-statements during initialization, instead variables assigned in \lstinline!when!-statements are initialized using \lstinline!v := pre(v)! before the body of the algorithm (since they are discrete), see \cref{execution-of-an-algorithm-in-a-model}.
 \end{nonnormative}
 
-Further constraints, necessary to determine the initial values of all variables, can be defined in the following ways:
+Further constraints, necessary to determine the initial values of all variables (depending on the variability of the variable, see \cref{component-variability-prefixes-discrete-parameter-constant} for definitions), can be defined in the following ways:
```

Comment:
I think we should try to stick with _component variability_ for consistency with how they were introduced:
```suggestion
Further constraints, necessary to determine the initial values of all variables (depending on the component variability, see \cref{component-variability-prefixes-discrete-parameter-constant} for definitions), can be defined in the following ways:
```